### PR TITLE
add HHFW wave simulation

### PR DESCRIPTION
**Pull request by sshiraiwa**
This pull add 
* image of HHFW wave simulation to gallery
* add short description to gallery.md

### CODE REVIEW

File: src/gallery.md [hunk 1]
```diff
@@ -174,6 +174,12 @@ This page collects screenshots from various simulations based on MFEM. Additiona
 
 *[Triple point shock interaction](https://computation.llnl.gov/projects/blast/triple-point-shock-interaction) on 4 elements of order 12.  Note the element curvature and the high variation of the field inside the lower right element.*
 
+---
+
+[![](img/gallery/nstxu_hhfw.png)](img/gallery/nstxu_hhfw.png)
+
+*Slice image of the high harmonic fast wave propagation in NSTX-U. Computed using MFEM 4th order H(curl) elements by [RF SciDAC project](https://www.rfscidac4.org).*
```
Comment (tzanio) [inline src/gallery.md:181]: Can you add a link or explanation what NSTX-U is :)

File: src/gallery.md [hunk 2]
```diff
@@ -174,6 +174,12 @@ This page collects screenshots from various simulations based on MFEM. Additiona
 
 *[Triple point shock interaction](https://computation.llnl.gov/projects/blast/triple-point-shock-interaction) on 4 elements of order 12.  Note the element curvature and the high variation of the field inside the lower right element.*
 
+---
+
+[![](img/gallery/nstxu_hhfw.png)](img/gallery/nstxu_hhfw.png)
```
Comment (tzanio) [inline src/gallery.md:179]: Do you happen to have a higher resolution image by any chance?

Comment (sshiraiwa) [reply src/gallery.md:179]: Let me generate another image, when the cluster comes back ;D

Comment (sshiraiwa) [reply src/gallery.md:179]: Added a new figure. Please take a look

Comment (mlstowell) [reply src/gallery.md:179]: That looks very sharp!

**Pull request merged**